README.md

```diff
@@ -1,3 +1,91 @@
 # clue
 
-Experimental GraphQL client.
\ No newline at end of file
+Experimental GraphQL client.
+
+## Usage
+
+### 1) Create a client
+
+Either:
+  * A `GraphQLClient[F[_], S]` (supporting queries and mutations), or
+  * A `GraphQLStreamingClient[F[_], S]` (supporting queries, mutations and subscriptions).
```

Comment:
Why would you use the non streaming client if it covers ball capabilities?

Reply:
The streaming client works on the premise that a bidirectional connection between client and server can be established and both sides can send messages arbitrarily.

There may be services that don't provide streaming, and therefore not expose a web socket endpoint (or other streaming capable mechanism). The "regular" client provides access to such "one shot" endpoints (eg: HTTP).